packages/venia-ui/lib/util/images.js

```diff
@@ -57,17 +57,22 @@ export const generateUrlFromContainerWidth = (
     );
 };
 
-export const generateSrcset = (imageURL, type) => {
+export const generateSrcset = (
+    imageURL,
+    type,
+    ratio
+) => {
     if (!imageURL || !type) return '';
 
+    ratio = ratio || DEFAULT_WIDTH_TO_HEIGHT_RATIO;
```

Comment:
It works but is not prescribed to re-assign arguments.

```suggestion
    const imageRatio = ratio || DEFAULT_WIDTH_TO_HEIGHT_RATIO;
```